db/migrate/20230418170053_inbound_api_configuration_permission.rb

```diff
@@ -0,0 +1,5 @@
+class InboundApiConfigurationPermission < ActiveRecord::Migration[6.1]
+  def change
+    add_column :roles, :can_manage_inbound_api_configurations, :boolean, default: false, null: false
```

Comment:
This should be done with `Role.ensure_permissions_exist` (you'll need a down that removes the columns I think.)  There should be plenty of examples.